test/base/test_objective.py

```diff
@@ -231,15 +143,15 @@ def test_finite_difference_checks():
 
     objective = pypesto.Objective(fun=fun, grad=grad)
 
-    def rel_err(eps):
+    def rel_err(eps_):
```

Comment:
why did you rename the variable to `eps_`? Looks unexpected to me :D

Reply:
just to avoid shadowing